rero_ils/modules/ebooks/utils.py

```diff
@@ -120,12 +126,18 @@ def update_document_holding(record, pid):
         reindex=True
     )
     for harvested_source in harvested_sources:
-        org = Organisation.get_record_by_online_harvested_source(
-            source=harvested_source['source'])
-        if org:
+        if org := Organisation.get_record_by_online_harvested_source(
+                source=harvested_source['source']):
             item_type_pid = org.online_circulation_category()
-            locations = org.get_online_locations()
-            for location_pid in locations:
+            location_pids = org.get_online_locations()
```

Comment:
This code seems similar than the previous one. Can you create a method for this?